apps/condo/migrations/20220721202935-0153_remove_ticket_placeclassifier_and_more.js

```diff
@@ -8,68 +8,68 @@ exports.up = async (knex) => {
 -- Remove field placeClassifier from ticket
 --
 SET CONSTRAINTS "Ticket_placeClassifier_01e3ec3a_fk_TicketPlaceClassifier_id" IMMEDIATE; ALTER TABLE "Ticket" DROP CONSTRAINT "Ticket_placeClassifier_01e3ec3a_fk_TicketPlaceClassifier_id";
```

Comment:
If we want to save `Ticket.placeClassifier` column, maybe we don't need to drop a constraint. It does not disturbs us, AFAIK.